samples/react-application-nav-search-driven/package.json

```diff
@@ -13,29 +13,29 @@
   },
   "dependencies": {
     "@fluentui/react": "8.106.4",
-    "@microsoft/decorators": "1.18.2",
+    "@microsoft/decorators": "1.20.0",
     "@microsoft/generator-sharepoint": "^1.19.0",
-    "@microsoft/sp-adaptive-card-extension-base": "1.18.2",
-    "@microsoft/sp-application-base": "1.18.2",
-    "@microsoft/sp-core-library": "1.18.2",
-    "@microsoft/sp-dialog": "1.18.2",
+    "@microsoft/sp-adaptive-card-extension-base": "1.20.0",
+    "@microsoft/sp-application-base": "1.20.0",
+    "@microsoft/sp-core-library": "1.20.0",
+    "@microsoft/sp-dialog": "1.20.0",
     "@types/react": "17.0.45",
     "@types/react-dom": "17.0.17",
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "tslib": "2.3.1"
   },
   "devDependencies": {
-    "@microsoft/eslint-config-spfx": "1.18.2",
-    "@microsoft/eslint-plugin-spfx": "1.18.2",
+    "@microsoft/eslint-config-spfx": "1.20.2",
+    "@microsoft/eslint-plugin-spfx": "1.20.2",
     "@microsoft/rush-stack-compiler-4.5": "0.5.0",
     "@microsoft/rush-stack-compiler-4.7": "0.1.0",
```

Comment:
we should leave only the rush stack 4.7. We may remove the older one

Reply:
I have removed the older version of rush stack. Please check lastest commit